architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,17 +175,101 @@ endif::[]
 [[image-streams]]
 == Image Streams
 
-An _image stream_ comprises any number of
-xref:../../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker-formatted
-container images] identified by tags. It presents a single virtual view of
-related images, similar to an image repository, and may contain images from any
-of the following:
+An _image stream_ is a reference to a xref:../../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker image].
+An image stream is simply a mapping between tags and actual images, and does not contain actual image data. It presents
+a single virtual view of related images, similar to an image repository.
 
-. Its own image repository in {product-title}'s
-xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[integrated
+// http://redhatgov.io/workshops/security_openshift/exercise1.3/
+An _image stream_ can automatically perform an action when new images are created. Builds and deployments can watch an image stream
+to receive notifications when new images are added and react by performing a build or deployment, respectively.
```

Comment:
the image stream isn't performing an action.  deploymentconfigs+buildconfigs are choosing to watch the imagestream for changes and then take some action if it does change.